pkgs/desktops/gnome-3/misc/evolution-ews/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchgit, gnome3, cmake, gettext, intltool, pkgconfig, evolution-data-server
+, sqlite, gtk3, webkitgtk, libgdata, ... }:
+
+stdenv.mkDerivation rec {
+  pname = "evolution-ews";
+  version = "3.36.3";
+  
+  src = fetchgit {
```

Comment:
Why not fetch it from GNOME mirrors?

https://github.com/NixOS/nixpkgs/blob/5bb1198cccb1a161e982dce84e18678ccec65529/pkgs/desktops/gnome-3/apps/evolution/default.nix#L49-L51

Reply:
done. now I'm fetching from gnome mirrors